packages/iTwinUI-react/src/core/Slider/Slider.test.tsx

```diff
@@ -23,6 +23,15 @@ const createBoundingClientRect = (
   toJSON: () => '',
 });
 
+// TODO: Is it needed to wrap all vertical aliders in this height: 500px <div>? Because without it, the slider has no height
+const verticalSliderWrapper = (verticalSlider: ReactNode) => {
+  return (
+    <div style={{ height: '500px', width: 'fit-content' }}>
+      {verticalSlider}
+    </div>
+  );
+};
```

Comment:
Since the slider expands to the height of the container, the slider would have no height if the container doesn't have an explicit height. Thus, I am wrapping all vertical sliders in a `<div>` with an explicit height. Is this alright? Or there is a better way?

Reply:
We talked yesterday that unit tests are rendered in mocked browser so no CSS or any styling is being taken into account. So you need to mock functions that return sizes or styling.

Reply:
**[DONE]** Right, thanks for teaching me about that. I removed `verticalSliderWrapper` as it was not needed since there is no CSS taken into account